src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * The List Widget View.
+ *
+ * @package Tribe\Events\Views\V2\Views\Widgets
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2\Views\Widgets;
+
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
+use Tribe__Context;
+use Tribe__Utils__Array as Arr;
+
+/**
+ * Class List_Widget_View
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Views\Widgets
+ */
+class Widget_List_View extends View {
+
+	/**
+	 * Slug for this view
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $slug = 'widget-list';
+
+	/**
+	 * Visibility for this view.
+	 *
+	 * @since TBD
+	 *
+	 * @var bool
+	 */
+	protected static $publicly_visible = false;
+
+	/**
+	 * Whether the View should display the events bar or not.
+	 *
+	 * @since TBD
+	 *
+	 * @var bool
+	 */
+	protected $display_events_bar = false;
+
+	/**
+	 * {@inheritDoc}
```

Comment:
:no_entry_sign: Doc comment for parameter "$context" missing

<sub>hash: e6c81f1dba87dbba0748c2266548589a</sub>